agent/statechange/statechange.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2014-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package statechange
```

Comment:
i didnt want to add anything to the ```agent/api``` that wasnt drectly related to api used to talk to the backend. right now it is the payload that is sent for the SubmitContainerStateChange and SubmitTaskStateChange APIs.